developerportal/apps/events/migrations/0031_event_sidebar.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.14 on 2020-07-06 15:49
```

Comment:
This files' changes are from another PR which was based off this one, which I should have held off merging, but didn't... Sorry.

Ignore for review